src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,726 @@
+        # NOTE in our case we don't have a prefix, thus we removed the "." from the keys later on!
+        dst_prefix: str = ""
+        src_prefix: str = "sem_seg_head.predictor"
+
+        renamed_keys = [
+            (f"{src_prefix}.class_embed.weight", f"{dst_prefix}class_predictor.weight"),
+            (f"{src_prefix}.class_embed.bias", f"{dst_prefix}class_predictor.bias"),
+        ]
+
+        mlp_len = 3
+        for i in range(mlp_len):
+            renamed_keys.extend(
+                [
+                    (f"{src_prefix}.mask_embed.layers.{i}.weight", f"{dst_prefix}mask_embedder.{i}.0.weight"),
+                    (f"{src_prefix}.mask_embed.layers.{i}.bias", f"{dst_prefix}mask_embedder.{i}.0.bias"),
+                ]
+            )
+        logger.info(f"Replacing keys {pformat(renamed_keys)}")
+        self.pop_all(renamed_keys, dst_state_dict, src_state_dict)
+
+    def convert(self, mask_former: MaskFormerModel) -> MaskFormerModel:
+        dst_state_dict = TrackedStateDict(mask_former.state_dict())
+        src_state_dict = self.original_model.state_dict()
+
+        self.replace_pixel_module(dst_state_dict, src_state_dict)
+        self.replace_transformer_module(dst_state_dict, src_state_dict)
+
+        logger.info(f"Missed keys are {pformat(dst_state_dict.diff())}")
+        logger.info(f"Not copied keys are {pformat(src_state_dict.keys())}")
+        logger.info("🙌 Done")
+
+        mask_former.load_state_dict(dst_state_dict)
+
+        return mask_former
+
+    def convert_instance_segmentation(
+        self, mask_former: MaskFormerForInstanceSegmentation
+    ) -> MaskFormerForInstanceSegmentation:
+        dst_state_dict = TrackedStateDict(mask_former.state_dict())
+        src_state_dict = self.original_model.state_dict()
+
+        self.replace_instance_segmentation_module(dst_state_dict, src_state_dict)
+
+        mask_former.load_state_dict(dst_state_dict)
+
+        return mask_former
+
+    @staticmethod
+    def using_dirs(checkpoints_dir: Path, config_dir: Path) -> Iterator[Tuple[object, Path, Path]]:
+        checkpoints: List[Path] = checkpoints_dir.glob("**/*.pkl")
+
+        for checkpoint in checkpoints:
+            logger.info(f"💪 Converting {checkpoint.stem}")
+            # find associated config file
+            config: Path = config_dir / checkpoint.parents[0].stem / "swin" / f"{checkpoint.stem}.yaml"
+
+            yield config, checkpoint
+
+
+def test(original_model, our_model: MaskFormerForInstanceSegmentation):
+    with torch.no_grad():
+
+        original_model = original_model.eval()
+        our_model = our_model.eval()
+
+        im = prepare_img()
+
+        tr = T.Compose(
+            [
+                T.Resize((384, 384)),
+                T.ToTensor(),
+                T.Normalize(
+                    mean=torch.tensor([123.675, 116.280, 103.530]) / 255.0,
+                    std=torch.tensor([58.395, 57.120, 57.375]) / 255.0,
```

Comment:
I guess this is the ImageNet default mean + standard deviation?

You can import this from image_utils.py: 

`from image_utils.py import IMAGE_DEFAULT_MEAN, IMAGE_DEFAULT_STD`